safe-global/web-core
# WIP: style: update illustrations + page headers

## What it solves

Update light/dark illustrations and add page headers

## How this PR fixes it

The app-wide illustrations have been updated to match the new theme in light/dark mode and page headers added, as described [here](https://5afe.slack.com/archives/C03DAGWJCR5/p1663773208255179).

## How to test it

The following should be visibly updated and different in light/dark mode:

- No pending transactions in the dashboard
- Spending limit settings (when none are set)
- Adding a custom app
- No NFTs (same image in light/dark mode)
- No address book entries found

The following pages should have interactable headers:

- Assets (with tabs and currency selector)
- Transactions (with tabs and batch execute/filter buttons respectively)
- Address book (with search, import, export and entry creation buttons)
- Safe Apps (as before)
- Settings (with tabs, replacing the sublist for categories)
